.github/workflows/e2e-local.yml

```diff
@@ -0,0 +1,96 @@
+name: E2E tests / Locally
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install common tools
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-commons.sh
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-solidity-node-modules
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+          
+      - name: Install Go
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: |
+          sudo rm -rf /usr/local/go
+          ./install-go.sh
+          echo "Using Go: $(go version)"
+
+      - name: Set GOPATH variable
+        run: echo "GOPATH=$HOME/go" >> $GITHUB_ENV
+
+      - name: Append PATH variable
+        run: |
+          echo "/usr/local/go/bin" >> $GITHUB_PATH
+          echo "$GOPATH/bin" >> $GITHUB_PATH
+         
+      - name: Install Geth
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-geth.sh
+
+      - name: Install Solidity
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-solidity.sh
+
+      - name: Install Protobuf
+        working-directory: ./deployments/local-setup-environment/provisioning/
+        run: ./install-protobuf.sh
+
+      - name: Initialize and run Geth
+        run: |
+          ls -l ethereum/data
+          ./initialize-geth.sh
+
+      - name: Run Bitcoin Core and ElectrumX
+        run: ./run-bitcoin.sh &
+
+      - name: Run install script
+        run: |
+          ls -l ethereum/data
+          ./run-geth.sh &
```

Comment:
Let's move it up to `Initialize and run Geth` step.

Reply:
Done, see 19f9bc71b0f3ebcaeba1d1eedbbcb53247c08246.